javalib/src/main/scala/java/lang/process/GenericProcessInfo.scala

```diff
@@ -0,0 +1,51 @@
+package java.lang.process
+
+import java.time.{Instant, Duration}
+import java.util.{Arrays => JArrays, Optional}
```

Comment:
Our style has been to use a short prefix for Java such as `import java.util as ju` and then use `ju.Arrays` for example.

Reply:
done